services/management/ia/rules_service.go

```diff
@@ -141,6 +143,9 @@ func (s *RulesService) CreateAlertRule(ctx context.Context, req *iav1beta1.Creat
 	if e != nil {
 		return nil, e
 	}
+
+	s.vmalert.RequestConfigurationUpdate()
```

Comment:
the ticket mentions these two points

```
Ensure that Alertmanager and VMAlert configuration reload (/-/reload API) is called from relevant IA APIs.
Ensure that Alertmanager's and VMAlert's supervisord configuration is reloaded from relevant IA APIs.
```
is the second point achieved using a call to `s.vmalert.RequestConfigurationUpdate()` ?

Reply:
No, but that particular method can't change VMAlert's supervisord configuration